### ISSUE TITLE
bug: API responses show user's hashed password

### ISSUE THREAD

**Issue by mariogarridopt** [mariogarridopt/bookit#15]
When returning user data, the API is showing hashed passwords like:

```JSON
{
	"id": 1,
	"username": "admin",
	"password": "8c6976e5b5410415bde908bd4dee15dfb167a9c873fc4bb8a81f6f2ab448a918",
	"active": 1,
	"role": "admin",
	"created_at": "2023-11-04T14:39:45",
	"updated_at": "2023-11-04T14:39:45"
}
```

This is not secure, let's remove those from the reply's.

**Comment by mariogarridopt** [comment]
Maybe we just delete the field before returning it to the response... I'm open to better ideas.

Before changes:

```python
@cross_origin()
@login_required
def get_my_data(user_id: int) -> dict:
    """Fetch user_id data"""
    user = Auth.select_first(id=user_id)
    return jsonify(user), 200
```

With changes:

```python
@cross_origin()
@login_required
def get_my_data(user_id: int) -> dict:
    """Fetch user_id data"""
    user = Auth.select_first(id=user_id)
    del user['password'] # remove password from response
    return jsonify(user), 200
```

But we have to repeat that code on multiple methods on the auth controller.

I'm open to discuss better solutions

**Comment by mariogarridopt** [comment]
We can make use of the `auth_service` and maybe create a function to sanitize auth data before sending to the response

**Comment by mariogarridopt** [comment]
fixed by #37
